app/src/main/java/ar/com/wolox/android/example/network/services/UserService.kt

```diff
@@ -0,0 +1,13 @@
+package ar.com.wolox.android.example.network.services
+
+import ar.com.wolox.android.example.model.User
+import okhttp3.RequestBody
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface UserService {
+
+    @POST("/auth/sign_in")
+    suspend fun signIn(@Body requestBody: RequestBody): Response<User>
```

Comment:
You don't need to use RequestBody for the body. Retrofit lets you use a custom data class with everything you need. Try creating a data class like "AuthenticationBody.kt" with the data you need to login and passing it here as a `@Body `annotation.